src/navigation/MainNavigator.tsx

```diff
@@ -14,6 +15,11 @@ export const MainNavigator = () => {
           component={Connections}
           options={{headerShown: false}}
         />
+        <Stack.Screen
+          name={'BookingScreen'}
+          component={() => <Booking />}
```

Comment:
As far as I know, react-navigation (at least v5) expects to receive the component directly instead of a callback. I think this way it won't really work since it doesn't have the navigation prop?
So I'd suggest using it as it appears in the docs!
```suggestion
          component={Booking}
```